source/dynk.f90

```diff
@@ -2501,7 +2468,7 @@ subroutine dynk_setvalue(element_name, att_name, newValue)
 
       case(40) ! Scatter
```

Comment:
This should probably be documented in Table 5.7/5.8 in the documentation of DYNK, `Table 5.7: Element types and attributes available in DYNK (part 1).`

Also, why is this table split in two and put at the end of the chapter containing lots of stuff (including but not limited to DYNK?) Table 5.6 does not have this issue...

Reply:
See issue #773 

Also, it seems to be documented, I just missed it in the confusion caused by #773 . Sorry 'bout that!